hw/top_earlgrey/data/ip/chip_aes_testplan.hjson

```diff
@@ -299,6 +302,7 @@
       stage: V2S
       si_stage: NA
       tests: ["chip_sw_aes_masking_off"]
+      bazel: []
```

Comment:
If the test point exist it means that a bazel target will eventually exist, which does not relates to `si_stage`, so I think that bazel should be mandatory and `!= []` when all the tests are implemented.

Reply:
As far as I'm aware, some tests will never have Bazel targets because they require mechanisms such as fault injection, e.g. `chip_sw_rv_core_ibex_alerts`.

Reply:
I see, so in that case bazel should not be mandatory.

Reply:
@moidx Could you let me know your opinion on this?

Reply:
I don't know of any top level test cases that won't have a set of binary targets. This is because of the need to initialize memories. The DV infrastructure is built in a way that even for tests like `chip_sw_rv_core_ibex_alerts`, it may be easier to just have a binary target.

However, we currently don't require bazel targets for simulation. Because of this, I am ok if we make this check conditional.